benchmarks/build.xml

```diff
@@ -69,14 +86,51 @@
 			<sysproperty key="external-properties" value="com/liferay/portal/tools/dependencies/portal-tools.properties" />
 			<sysproperty key="sample-sql-properties" value="${sample.sql.properties.file}" />
 		</java>
+
+		<concat append="true" destfile="${sample-data-sql}">
+			<filelist dir="../sql/portal/" files="portal-${sample.sql.db.type}.sql" />
+			<filelist dir="../sql/indexes/" files="indexes-${sample.sql.db.type}.sql" />
+		</concat>
+
+		<for list="${portal.module.dir.list}" param="module">
+			<sequential>
+				<build-sql dir="@{module}" />
+			</sequential>
+		</for>
 	</target>
 
 	<target name="clean">
 		<delete includeemptydirs="true">
 			<fileset
 				dir="."
-				includes="*.csv,*.sql,benchmarks-actual.properties,output/**,temp/**"
+				includes="*.csv,*.sql,benchmarks-actual.properties,liferay/**,output/**,temp/**"
 			/>
 		</delete>
 	</target>
+
+	<target depends="clean, build-sample-sql" name="merge-sql">
```

Comment:
@slnn Who is going to use this "merge-sql"?
If I want to run sample sql builder to get a sample sql file to import into my database, what should I do?